js/apps/admin-ui/src/user/UserCredentials.tsx

```diff
@@ -329,6 +331,28 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
     }
   };
 
+  const useFederatedCredentials = user.federationLink || user.origin;
+  const [credentialTypes, setCredentialTypes] = useState<string[]>([]);
+
+  if (useFederatedCredentials) {
+    useFetch(
```

Comment:
Hook functions are [not allowed](https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks) to be called conditionally.